src/utils/setup/save-asset-file.js

```diff
@@ -26,17 +26,17 @@ const saveAssetFile = async asset => {
 };
 
 // Write all assets to static directory
-const saveAssetFiles = async (assets, stitchClient) => {
+const saveAssetFile = async (asset, stitchClient) => {
```

Comment:
We hit stitch more often now here since we call this once per asset. This was actually faster than trying to memoize all of the assets in stitch and then filtering one out and writing out on demand.

Reply:
That seems like it could be a lot. What's the delta in the number of calls to the network from our current build to this one?

Reply:
It's going from one to ~252. I'm going to try to leverage the in-memory assets array we had before to try and move this back down to one

Reply:
Thanks, let me know how that goes - I think making that many calls is asking for trouble (and could grow exponentially over time)

Reply:
We're back down to one stitch call now?

Reply:
yes we are!